testkit/src/main/java/org/apache/calcite/test/CalciteAssert.java

```diff
@@ -205,6 +205,10 @@ private CalciteAssert() {}
           return this;
         }
 
+        @Override public AssertThat with(SqlConformanceEnum conformance) {
```

Comment:
what happens here?

Reply:
Same as the previous one, we need to set the conformance props to control whether the parameterless function can use parentheses.